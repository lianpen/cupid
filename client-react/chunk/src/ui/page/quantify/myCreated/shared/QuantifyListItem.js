/*
 * @Author: lianpen
 * @Date:   2018-06-07 17:16:12
 */
import {
    List
} from 'antd-mobile'

import constants from 'global/constants'
import dateUtil from 'util/date'
import Svg_reward_wait from 'assets/svg/md_wait.svg'
import Svg_reward_reject from 'assets/svg/md_reject.svg'
import Svg_reward_draw from 'assets/svg/md_draw.svg'
import Svg_reward_pass from 'assets/svg/md_pass.svg'
import Svg_reward_del from 'assets/svg/md_del.svg'
import Svg_wait from 'assets/svg/sm_wait.svg'
import Svg_pass from 'assets/svg/sm_pass.svg'
import Svg_draft from 'assets/svg/sm_draft.svg'
import Svg_sm_del from 'assets/svg/sm_del.svg'
import moment from 'moment'
import Svg_reject from 'assets/svg/lg_reject.svg'
import Svg_draw from 'assets/svg/lg_draw.svg'
import Svg_del from 'assets/svg/lg_del.svg'

export default class QuantifyListItem extends BaseRc {
    render() {
        const quantify = this.props.quantify
        return (
            <List.Item multipleLine onClick={ this.props.onClick } className="shared-list" >
                    <div className="shared-list-lf" style={{ top: 7 }}>
                         { this.renderRewardSvg(quantify) }
                    </div>
                    <div className="shared-list-bd title">{ quantify.data.quantifyName }</div>
                    { this.renderAscore(quantify) }
                    { this.renderBscore(quantify) }
                    { this.renderOp(quantify) }
                    { this.renderPersonNum(quantify) }
                    <List.Item.Brief>
                         <span className="shared-list-fd">
                             <label>初审人</label>:&nbsp;
                             { quantify.data.attnFullName }
                         </span>
                         </List.Item.Brief>
                         <List.Item.Brief>
                         <span className="shared-list-fd">
                             <label>终审人</label>:&nbsp;
                             { quantify.data.auditFullName }
                         </span>
                    </List.Item.Brief>
                    <List.Item.Brief>
                         { this.renderCheckStatusBottom(quantify) }
                    </List.Item.Brief>
                    <div className="shared-list-operate">
                         { this.renderOperateSvg(quantify) }
                    </div>
               </List.Item>
        )
    }

    /**
     * 渲染左侧审核流程图标
     * 1.正常流程待初审和待终审显示等待图标
     */
    renderRewardSvg(quantify) {
        const status = parseInt(quantify.data.status)
        const operate = quantify.data.operate

        if (status === constants.CHECK_STATUS.PASSED) {
            if (quantify.data.isDel) {
                return <Svg_reward_del width={ 26 } height= { 26 } className='text-red' />
            }
            return <Svg_reward_pass width={ 26 } height= { 26 } className='text-primary' />
        } else {
            if (operate === 1) {
                return <Svg_reward_reject width={ 26 } height= { 26 } className='text-red' />
            } else if (operate === 2) {
                return <Svg_reward_draw width={ 26 } height= { 26 } className='text-yellow' />
            } else {
                return <Svg_reward_wait width={ 26 } height= { 26 } className='text-primary' />
            }
        }
    }

    /**
     * 计算A分b分
     */
    getScore(reward, deduct) {
        if (reward && deduct) {
            return (
                <span>
                    { reward } / <span className="text-red">-{ deduct }</span> 
                </span>
            )
        } else if (reward && !deduct) {
            return reward
        } else if (!reward && deduct) {
            return (
                <span className="text-red">-{ deduct }</span>
            )
        } else {
            return null
        }
    }

    /**
     * 渲染a分
     */
    renderAscore(quantify) {
        if (!quantify || !quantify.data || !quantify.data.integralQuantifyInfo) return null
        const ascoreReward = quantify.data.integralQuantifyInfo.ascoreReward
        const ascoreDeduct = quantify.data.integralQuantifyInfo.ascoreDeduct
        if (!ascoreReward && !ascoreDeduct) {
            return null
        }
        return (
            <List.Item.Brief>
               <span className="shared-list-bd">
                    <label>A分</label>:&nbsp;
                    { this.getScore(ascoreReward, ascoreDeduct) }
               </span>
            </List.Item.Brief>
        )
    }

    /**
     * 渲染b分
     */
    renderBscore(quantify) {
        if (!quantify || !quantify.data || !quantify.data.integralQuantifyInfo) return null
        const bscoreReward = quantify.data.integralQuantifyInfo.bscoreReward
        const bscoreDeduct = quantify.data.integralQuantifyInfo.bscoreDeduct
        if (!bscoreReward && !bscoreDeduct) {
            return null
        }
        return (
            <List.Item.Brief>
                    <span className="shared-list-bd">
                         <label>B分</label>:&nbsp;
                         { this.getScore(bscoreReward, bscoreDeduct) }
                    </span>
               </List.Item.Brief>
        )
    }
    /**
     * 渲染产值
     */
    renderOp(quantify) {
        if (!quantify || !quantify.data || !quantify.data.integralQuantifyInfo) return null
        const opscoreGen = quantify.data.integralQuantifyInfo.opscoreGen
        const opscoreReal = quantify.data.integralQuantifyInfo.opscoreReal
        const opscoreVirtual = quantify.data.integralQuantifyInfo.opscoreVirtual
        if (!opscoreGen && !opscoreReal && !opscoreVirtual) {
            return null
        }
        return (
            <List.Item.Brief>
                    <span className="shared-list-bd">
                         <label>虚/实/创</label>:&nbsp;
                         { opscoreVirtual+'/'+opscoreReal+'/'+ opscoreGen}
                    </span>
               </List.Item.Brief>
        )
    }

    /**
     * 渲染人次
     */
    renderPersonNum(quantify) {
        const personNum = quantify.data.integralQuantifyInfo ? quantify.data.integralQuantifyInfo.empCount : 0
        if (!personNum) {
            return null
        }
        return (
            <List.Item.Brief>
                <span className="shared-list-bd">
                    <label>人次</label>:&nbsp;
                    { personNum }
                </span>
            </List.Item.Brief>
        )
    }

    /**
     * 渲然审核状态 底部
     */
    renderCheckStatusBottom(quantify) {
        const status = quantify.data.status
        const operate = quantify.data.operate
        const isDel = quantify.data.isDel
        if (isDel == 1) {
            return (
                <span className="shared-list-st">
                     <span className="text-red">
                         <Svg_sm_del width={ 16 } height= { 16 } />
                         &nbsp;
                         <span className='text-red'>已作废</span>
                     </span>
                     <span>{ new moment(quantify.data.lastEditDate).format('YYYY-MM-DD') }</span>
                </span>

            )
        } else if (status === '0') {
            let spanClassName = '';
            if (operate === 1) {
                spanClassName = 'text-red'
            } else if (operate === 2) {
                spanClassName = 'text-yellow'
            } else {
                spanClassName = 'text-primary'
            }
            return (
                <span className="shared-list-st">
                     <span className={ spanClassName }>
                         <Svg_draft width={ 14 } height= { 14 } />
                         &nbsp;拟稿
                     </span>
                     <span >
                          {new moment(quantify.lastEditDate).format('YYYY-MM-DD') }
                     </span>
                </span>
            )

        } else if (status === '1') {
            return (
                <span className="shared-list-st">
                     <span className="text-primary">
                         <Svg_wait width={ 14 } height= { 14 } />
                         &nbsp;待初审
                     </span>
                     <span >
                          { new moment(quantify.data.createDate).format('YYYY-MM-DD') }
                     </span>
                </span>
            )
        } else if (status === '2') {
            return (
                <span className="shared-list-st">
                    <span className="text-primary">
                        <Svg_wait width={ 14 } height= { 14 } className='text-primary' />
                        &nbsp;待终审
                    </span>
                    <span >{ new moment(quantify.data.attnTime).format('YYYY-MM-DD') }</span>
                </span>
            )
        } else if (status === '3') {
            return (
                <span className="shared-list-st">
                    <span className="text-primary">
                        <Svg_pass width={ 14 } height= { 14 } className='text-primary' />
                        &nbsp;审核通过
                    </span>
                    <span>{ new moment(quantify.data.auditTime).format('YYYY-MM-DD') }</span>
                </span>
            )
        }
    }

    /**
     * 渲染驳回驳回状态
     */
    renderOperateSvg(quantify) {
        const operate = quantify.data.operate
        if (quantify.data.isDel) {
            return <Svg_del  width={ 60 } height={ 60 } className="text-red" />
        }
        if (operate === 1) {
            return <Svg_reject width={ 60 } height={ 60 } className="text-red" />
        } else if (operate === 2) {
            return <Svg_draw width={ 60 } height={ 60 } className="text-yellow" />
        }
        return null
    }
}