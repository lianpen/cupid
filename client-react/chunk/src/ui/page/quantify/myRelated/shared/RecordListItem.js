/*
 * @Author: lianpen
 * @Date:   2018-06-07 17:16:12
 */
import {
    List
} from 'antd-mobile'

import constants from 'global/constants'
import dateUtil from 'util/date'
import Svg_reward_wait from 'assets/svg/md_wait.svg'
import Svg_reward_reject from 'assets/svg/md_reject.svg'
import Svg_reward_draw from 'assets/svg/md_draw.svg'
import Svg_reward_pass from 'assets/svg/md_pass.svg'
import Svg_reward_del from 'assets/svg/md_del.svg'
import Svg_wait from 'assets/svg/sm_wait.svg'
import Svg_pass from 'assets/svg/sm_pass.svg'
import Svg_draft from 'assets/svg/sm_draft.svg'
import Svg_sm_del from 'assets/svg/sm_del.svg'
import moment from 'moment'
import Svg_reject from 'assets/svg/lg_reject.svg'
import Svg_draw from 'assets/svg/lg_draw.svg';
import Svg_del from 'assets/svg/lg_del.svg'

export default class RecordListItem extends BaseRc {
    render() {
        const record = this.props.record
        const quantify = record.quantify
        return (
            <List.Item multipleLine onClick={ this.props.onClick } className="shared-list" >
                    <div className="shared-list-lf" style={{ top: 7 }}>
                        { this.renderRewardSvg(record) }
                    </div>
                    <div className="shared-list-bd title">{ '事件： ' + record.eventName }</div>
                    { this.renderAscore(record) }
                    { this.renderBscore(record) }
                    { this.renderOp(record) }
                    <List.Item.Brief>
                        <span className="shared-list-fd">
                            <label>奖扣时间</label>:&nbsp;
                            { this.getTime(record.quantify.quantifyDate) }
                        </span>
                    </List.Item.Brief>
                    <List.Item.Brief>
                         <span className="shared-list-fd">
                              <label>初审人</label>:&nbsp;
                              { record.quantify.attnFullName || record.attnFullName }
                         </span>
                    </List.Item.Brief>
                    <List.Item.Brief>
                         <span className="shared-list-fd">
                              <label>终审人</label>:&nbsp;
                              { record.quantify.auditFullName || record.auditFullName }
                         </span>
                    </List.Item.Brief>
                    <List.Item.Brief>
                         { this.renderCheckStatusBottom(record) }
                    </List.Item.Brief>
                    <div className="shared-list-operate">
                         { this.renderOperateSvg(record) }
                    </div>
               </List.Item>
        )
    }

    /**
     * 渲染左侧审核流程图标
     * 1.正常流程待初审和待终审显示等待图标
     */
    renderRewardSvg(record) {
        const quantify = record.quantify
        const status = parseInt(quantify.status)
        const operate = quantify.operate

        if (status === constants.CHECK_STATUS.PASSED) {
            if (record.isDel) {
                return <Svg_reward_del width={ 26 } height= { 26 } className='text-red' />
            }
            return <Svg_reward_pass width={ 26 } height= { 26 } className='text-primary' />
        } else {
            if (operate === 1) {
                return <Svg_reward_reject width={ 26 } height= { 26 } className='text-red' />
            } else if (operate === 2) {
                return <Svg_reward_draw width={ 26 } height= { 26 } className='text-yellow' />
            } else {
                return <Svg_reward_wait width={ 26 } height= { 26 } className='text-primary' />
            }
        }
    }

    /**
     * 计算A分b分
     */
    getScore(score) {
        score = parseInt(score)
        if (isNaN(score)) score = 0
        if (score >= 0) {
            return score
        } else {
            return (
                <span className="text-red">{ score }</span>
            )
        }
    }

    /**
     * 渲染a分
     */
    renderAscore(record) {
        const score = record.ascore
        if (!score) {
            return null
        }
        return (
            <List.Item.Brief>
                    <span className="shared-list-bd">
                         <label>A分</label>:&nbsp;
                         { this.getScore(score) }
                    </span>
               </List.Item.Brief>
        )
    }

    /**
     * 渲染b分
     */
    renderBscore(record) {
        const score = record.bscore
        if (!score) {
            return null
        }
        return (
            <List.Item.Brief>
                    <span className="shared-list-bd">
                         <label>B分</label>:&nbsp;
                         { this.getScore(score) }
                    </span>
               </List.Item.Brief>
        )
    }
    /**
     * 渲染产值
     */
    renderOp(record) {
        const type = record.opType
        const score = record.opScore
        const virtual = type == constants.OUTPUT_TYPE.VIRTUAL ? score : 0
        const real = type == constants.OUTPUT_TYPE.REAL ? score : 0
        const create = type == constants.OUTPUT_TYPE.CREATE ? score : 0
        if (!virtual && !real && !create) {
            return null
        }
        return (
            <List.Item.Brief>
                    <span className="shared-list-bd">
                         <label>虚/实/创</label>:&nbsp;
                         { virtual+'/'+real+'/'+create }
                    </span>
               </List.Item.Brief>
        )
    }

    /**
     * 渲然审核状态 底部
     */
    renderCheckStatusBottom(record) {
        const quantify = record.quantify
        const status = quantify.status
        const operate = quantify.operate
        const isDel = record.isDel
        if (isDel == 1) {
            return (
                <span className="shared-list-st">
                     <span className="text-red">
                         <Svg_sm_del width={ 16 } height= { 16 } />
                         &nbsp;
                         <span className='text-red'>已作废</span>
                     </span>
                </span>

            )
        } else if (status === '0') {
            let spanClassName = '';
            if (operate === 1) {
                spanClassName = 'text-red'
            } else if (operate === 2) {
                spanClassName = 'text-yellow'
            } else {
                spanClassName = 'text-primary'
            }
            return (
                <span className="shared-list-st">
                     <span className={ spanClassName }>
                         <Svg_draft width={ 14 } height= { 14 } />
                         &nbsp;拟稿
                     </span>
                </span>
            )

        } else if (status === '1') {
            return (
                <span className="shared-list-st">
                         <span className="text-primary">
                             <Svg_wait width={ 14 } height= { 14 } />
                             &nbsp;待初审
                         </span>
                    </span>
            )
        } else if (status === '2') {
            return (
                <span className="shared-list-st">
                    <span className="text-primary">
                        <Svg_wait width={ 14 } height= { 14 } className='text-primary' />
                        &nbsp;待终审
                    </span>
                </span>
            )
        } else if (status === '3') {
            return (
                <span className="shared-list-st">
                    <span className="text-primary">
                        <Svg_pass width={ 14 } height= { 14 } className='text-primary' />
                        &nbsp;审核通过
                    </span>
                </span>
            )
        }
    }

    /**
     * 计算时间
     */
    getTime(str) {
        if (!str) return ''
        return new moment(str).format('YYYY-MM-DD')
    }

    /**
     * 渲染驳回驳回状态
     */
    renderOperateSvg(record) {
        const quantify = record.quantify
        const operate = quantify.operate
        if (record.isDel) {
            return <Svg_del width={ 60 } height={ 60 } className="text-red" />
        }
        if (operate === 1) {
            return <Svg_reject width={ 60 } height={ 60 } className="text-red" />
        } else if (operate === 2) {
            return <Svg_draw width={ 60 } height={ 60 } className="text-yellow" />
        }
        return null
    }
}