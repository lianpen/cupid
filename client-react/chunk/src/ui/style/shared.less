body {
    font-family: -apple-system-font, "Helvetica Neue", "PingFang SC", "Hiragino Sans GB", "Microsoft YaHei", "Noto Sans CJK", sans-serif, pingfang, tahoma, arial;
}

.mt10 {
    margin-top: 10px;
}
.shared-dropMenu {
    text-align: center;
    line-height: 3;
    color: #666;
    background: #fff;
    &-item {
        svg {
            vertical-align: -2px;
            margin-left: 4px;
        }
    }
}

.shared-menu {
	position: absolute;
  	z-index: 90 !important;
  	width: 100%;
    &-more {
        position: absolute;
        width: 100%;
        z-index: 100;
        background: #fff;
        padding: 5px 10px;
        bottom: -33px;
        text-align: center;
    }
}

.shared-menu-mask {
	position: fixed;
  	top: 0;
	width: 100%;
  	height: 100%;
  	background-color: #000;
  	opacity: 0.4;
  	z-index: 89;	
}

.shared-bottomSubmit {
    position: fixed !important;
    z-index: 99;
    bottom: 0;
    left: 0;
    right: 0;
    padding: 12px 15px;
    background: #fff;
    display: flex;
    .am-button {
        flex: 1;
    }
    &-two_left {
        margin-right: 7px;
    }
    &-two_right {
        margin-left: 7px;
    }
}

.shared-underNavbar {
    margin-top: 44px;
    border-bottom: 1px solid #ddd;
}

.shared-border-top {
    position: relative;
    &:before {
        content: '';
        position: absolute;
        background-color: #ddd;
        display: block;
        z-index: 1;
        top: 0;
        right: auto;
        bottom: auto;
        left: 0;
        width: 100%;
        height: 1px;
        transform-origin: 50% 100%;
        transform: scaleY(0.5);
    }
}

.shared-border-bottom {
    position: relative;
    &:after {
        content: '';
        position: absolute;
        background-color: #ddd;
        display: block;
        z-index: 1;
        top: auto;
        right: auto;
        bottom: 0;
        left: 0;
        width: 100%;
        height: 1PX;
        transform-origin: 50% 50%;
        transform: scaleY(0.5);
    }
}

.shared-person {
    display: flex;
    align-items: center;
    // padding: 4px 0;
    &-circle {
        width: 32px;
        height: 32px;
        line-height: 32px;
        text-align: center;
        background: #00a5e0;
        color: #fff;
        border-radius: 50%;
        font-size: 13px;
    }
    &-bd {
        margin-left: 20px;
        flex: 1;
        h3 {
            font-weight: normal;
            font-size: 15px;
        }
        p {
            color: #a1a1a1;
            font-size: 12px;
        }
    }
}

.shared-person-circle2 {
    width: 24px;
    height: 24px;
    line-height: 24px;
    text-align: center;
    background: #00a5e0;
    color: #fff;
    border-radius: 50%;
    font-size: 12px;
    margin-top: 4px;
}

.shared-link {
    margin-top: 10px;
}
.shared-refresh-text {
    padding: 30px;
    display: flex;
    justify-content: center;
    align-items: center;
}
.sharf-fixed-box {
    position: fixed;
    z-index: 99;
    background-color: #f5f5f5;
    top: 45px;
    left: 0;
    right: 0;
}