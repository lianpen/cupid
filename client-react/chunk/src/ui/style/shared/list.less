.shared-list {
	&.active {
		.am-list-content {
			color: #4a7ec0;
			>span {
				color: #4a7ec0;
			}
		}
		.am-list-extra {
			color: #4a7ec0;
		}
	}
	.am-list-content {
		padding-left: 30px;
		position: relative;
		.am-list-brief {
			margin-top: 0;
		}
	}
	&-lf {
		position: absolute;
		left: 0;
	}
	&-hd {
		font-size: 15px;
		color: #a1a1a1;
		.am-badge {
			float: right;
			.am-badge-text {
				margin-right: 0
			}
		}
	}
	&-bd {
		&.title {
			font-size: 15px;
			white-space: normal;
			display: block;
    		margin: 10px 0;
		}
	    font-size: 12px;
	    color: #323232;
		line-height: 1;
		label {
			display: inline-block;
		    width: 56px;
		    text-align: justify;
		    text-align-last: justify;
		    -khtml-text-align-last: justify;
		    -moz-text-align-last: justify;
		    text-align: right;
		}
	}
	&-fd {
		font-size: 12px;
		color: #a1a1a1;
		line-height: 1;
		label {
			display: inline-block;
		    width: 56px;
		    text-align: justify;
		    text-align-last: justify;
		    text-align: right;
		}
	}
	&-st {
		display: flex;
		justify-content: space-between;
		align-items: center;
		font-size: 12px;
		margin-top: 5px;
		> span {
			svg {
				vertical-align: text-bottom;
			}
		}
	}
	&-operate {
		display: inline-block;
		position: absolute;
    	right: 3px;
    	bottom: 11px;
	}
}