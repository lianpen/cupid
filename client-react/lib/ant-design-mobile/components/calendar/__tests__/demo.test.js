import { webDemoTest } from '../../../tests/shared/demoTest';

webDemoTest('calendar');
