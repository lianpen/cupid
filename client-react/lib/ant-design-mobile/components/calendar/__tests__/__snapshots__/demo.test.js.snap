// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/calendar/demo/basic.md correctly 1`] = `
<div>
  <div
    class="am-list calendar-list"
    style="background-color:white"
  >
    <div
      class="am-list-body"
    >
      <div
        class="am-list-item item am-list-item-middle"
      >
        <div
          class="am-list-line"
        >
          <div
            class="am-list-content"
          >
            中文
          </div>
          <div
            class="am-list-extra"
          >
            <label
              class="am-switch right"
            >
              <input
                checked=""
                class="am-switch-checkbox"
                name=""
                type="checkbox"
                value="on"
              />
              <div
                class="checkbox"
              />
            </label>
          </div>
        </div>
        <div
          class="am-list-ripple"
          style="display:none"
        />
      </div>
      <div
        class="am-list-item am-list-item-middle"
      >
        <div
          class="am-list-line"
        >
          <div
            class="am-list-content"
          >
            选择日期区间
          </div>
          <div
            aria-hidden="true"
            class="am-list-arrow am-list-arrow-horizontal"
          />
        </div>
        <div
          class="am-list-ripple"
          style="display:none"
        />
      </div>
      <div
        class="am-list-item am-list-item-middle"
      >
        <div
          class="am-list-line"
        >
          <div
            class="am-list-content"
          >
            选择日期时间区间
          </div>
          <div
            aria-hidden="true"
            class="am-list-arrow am-list-arrow-horizontal"
          />
        </div>
        <div
          class="am-list-ripple"
          style="display:none"
        />
      </div>
      <div
        class="am-list-item am-list-item-middle"
      >
        <div
          class="am-list-line"
        >
          <div
            class="am-list-content"
          >
            选择日期
          </div>
          <div
            aria-hidden="true"
            class="am-list-arrow am-list-arrow-horizontal"
          />
        </div>
        <div
          class="am-list-ripple"
          style="display:none"
        />
      </div>
      <div
        class="am-list-item am-list-item-middle"
      >
        <div
          class="am-list-line"
        >
          <div
            class="am-list-content"
          >
            选择日期时间
          </div>
          <div
            aria-hidden="true"
            class="am-list-arrow am-list-arrow-horizontal"
          />
        </div>
        <div
          class="am-list-ripple"
          style="display:none"
        />
      </div>
      <div
        class="am-list-item am-list-item-middle"
      >
        <div
          class="am-list-line"
        >
          <div
            class="am-list-content"
          >
            选择日期区间(快捷)
          </div>
          <div
            aria-hidden="true"
            class="am-list-arrow am-list-arrow-horizontal"
          />
        </div>
        <div
          class="am-list-ripple"
          style="display:none"
        />
      </div>
      <div
        class="am-list-item am-list-item-middle"
      >
        <div
          class="am-list-line"
        >
          <div
            class="am-list-content"
          >
            选择日期时间区间(快捷)
          </div>
          <div
            aria-hidden="true"
            class="am-list-arrow am-list-arrow-horizontal"
          />
        </div>
        <div
          class="am-list-ripple"
          style="display:none"
        />
      </div>
      <div
        class="am-list-item am-list-item-middle"
      >
        <div
          class="am-list-line"
        >
          <div
            class="am-list-content"
          >
            大行距
          </div>
          <div
            aria-hidden="true"
            class="am-list-arrow am-list-arrow-horizontal"
          />
        </div>
        <div
          class="am-list-ripple"
          style="display:none"
        />
      </div>
      <div
        class="am-list-item am-list-item-middle"
      >
        <div
          class="am-list-line"
        >
          <div
            class="am-list-content"
          >
            不无限滚动
          </div>
          <div
            aria-hidden="true"
            class="am-list-arrow am-list-arrow-horizontal"
          />
        </div>
        <div
          class="am-list-ripple"
          style="display:none"
        />
      </div>
      <div
        class="am-list-item am-list-item-middle"
      >
        <div
          class="am-list-line"
        >
          <div
            class="am-list-content"
          >
            水平进入
          </div>
          <div
            aria-hidden="true"
            class="am-list-arrow am-list-arrow-horizontal"
          />
        </div>
        <div
          class="am-list-ripple"
          style="display:none"
        />
      </div>
      <div
        class="am-list-item am-list-item-middle"
      >
        <div
          class="am-list-line"
        >
          <div
            class="am-list-content"
          >
            默认选择范围
          </div>
          <div
            aria-hidden="true"
            class="am-list-arrow am-list-arrow-horizontal"
          />
        </div>
        <div
          class="am-list-ripple"
          style="display:none"
        />
      </div>
      <div
        class="am-list-item am-list-item-middle"
      >
        <div
          class="am-list-line"
        >
          <div
            class="am-list-content"
          >
            onSelect API
          </div>
          <div
            aria-hidden="true"
            class="am-list-arrow am-list-arrow-horizontal"
          />
        </div>
        <div
          class="am-list-ripple"
          style="display:none"
        />
      </div>
    </div>
  </div>
  <div
    class="am-calendar"
  >
    <span>
      <div
        class="mask animate"
        style="display:none"
      />
    </span>
    <span>
      <div
        class="content animate"
        style="display:none"
      />
    </span>
  </div>
</div>
`;
